tensorboard/webapp/widgets/line_chart_v2/line_chart_demo_component.ts

```diff
@@ -0,0 +1,111 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {ChangeDetectionStrategy, Component} from '@angular/core';
+
+import {DataSeries, DataSeriesMetadataMap, ScaleType} from './lib/public_types';
+
+const NUM_SERIES = 5;
+const NUM_POINTS = 1000;
+const NUM_PERIODS = 2;
+
+function createData(numPointsPerSeries: number, lambda: number): DataSeries {
+  return {
+    id: `sine_${lambda}`,
+    points: [...new Array(numPointsPerSeries)].map((_, index) => {
+      return {
+        x: index,
+        y:
+          Math.sin(
+            (lambda * index * 2 * Math.PI * NUM_PERIODS) / numPointsPerSeries
```

Comment:
nit: `lambda` suggests a wavelength, but this is more like a frequency, no?

Or is this `lambda` consistent with some usage that I don’t know?

Reply:
Oh that's what you meant. I meant to use it as a constant like alpha but now I see how it is very confusing given 𝝀 literally mean wave length in physics sense. 

TBH, this method used to take a lambda function to return `y` given `x` and I was too lazy to change the name :)